spec/gutter-component-spec.coffee

```diff
@@ -0,0 +1,40 @@
+GutterComponent = require '../src/gutter-component'
+React = require 'react-atom-fork'
+
+HIDDEN_CLASS = 'hidden'
+
+describe 'GutterComponent visibility tests', ->
```

Comment:
I think the root `describe` should be `GutterComponent`, then have nested `describe`s for its attributes.